lib/spdx/cabal.nix

```diff
@@ -0,0 +1,12 @@
+lib:
+license: let
+  license-map = import ../cabal-licenses.nix lib;
+  otherLicenseWarning = lic:
+    builtins.trace "WARNING: license \"${lic}\" not found"
+                   license-map.LicenseRef-OtherLicense;
+  spdx = import ../lib/spdx/parser.nix;
+  licenses = spdx.compoundExpression license;
+in if licenses == []
+   then otherLicenseWarning license
+   else map (lic: license-map.${lic} or (otherLicenseWarning lic))
+            (lib.unique (head licenses)._1);
```

Comment:
I think the `;` at the end of this file is causing this:
```
hydra-eval-jobs returned exit code 1:
error: "syntax error, unexpected ';', expecting $end, at \u001b[1m/nix/store/yxqx5sichwiwvgpva3ncyxkg6vvlxv44-source/lib/spdx/cabal.nix\u001b[0m:12:44"
error: unexpected EOF reading a line
```

Reply:
😳 
This time around I at least loaded most of the files in the nix repl so there shouldn't be syntax errors like this.
Didn't check the builders because I didn't want to pass all the arguments. I could probably use callPackage though?